WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -786,13 +824,45 @@ public ErrorDescriptors ValidateAddress()
 			return new ErrorDescriptors(new ErrorDescriptor(ErrorSeverity.Error, "Invalid address."));
 		}
 
-		[ValidateMethod(nameof(ValidateAddress))]
+		public ErrorDescriptors ValidateCustomChangeAddress()
+		{
+			if (string.IsNullOrWhiteSpace(CustomChangeAddress))
+			{
+				return ErrorDescriptors.Empty;
+			}
+
+			if (IsMax)
+			{
+				return new ErrorDescriptors(new ErrorDescriptor(ErrorSeverity.Error, "Spending whole coins does not generate change."));
+			}
```

Comment:
I think this can never happen.
If `IsMax` is true then `CustomChangeAddress` control is not enabled.

Reply:
It can never happen. Yet we leave it here in case someone ruins something in the future.